src/preload.ts

```diff
@@ -30,34 +24,14 @@ function makeKiosk(): KioskBrowser.Kiosk {
       )) as OpenDialogReturnValue;
     },
 
-    async showSaveDialog(
-      options?: SaveDialogOptions,
-    ): Promise<SaveDialogReturnValue> {
-      debug('forwarding `showSaveDialog` to main process');
-      return (await ipcRenderer.invoke(
-        showSaveDialogChannel,
-        options,
-      )) as SaveDialogReturnValue;
-    },
-
     async log(message: string): Promise<void> {
       debug('forwarding `log` to the main process');
       await ipcRenderer.invoke(logChannel, message);
     },
 
-    async saveAs(
-      options?: PromptToSaveOptions,
-    ): Promise<FileWriter | undefined> {
-      return await fromPrompt(options);
-    },
-
-    quit(exitCode?: number): void {
+    quit(): void {
       debug('forwarding `quit` to main process');
-      if (typeof exitCode === 'undefined') {
-        void ipcRenderer.invoke(quitChannel);
-      } else {
-        void ipcRenderer.invoke(quitChannel, exitCode);
-      }
+      void ipcRenderer.invoke(quitChannel);
     },
 
     async captureScreenshot(): Promise<Buffer> {
```

Comment:
do we need to change the return type from `Buffer` to `Uint8Array` here as well?